### ISSUE TITLE
Mention ad blockers in documentation

### ISSUE THREAD

**Issue by gschlabitz** [css-modules/css-modules#159]
I was testing an app and found that all my labels where missing on my mobile browser when they showed up fine on the desktop. After some remote debugging, I found that [an ad blocker](https://www.purify-app.com/) was identifying one of the generated classnames, `App__label___3PDAd`, as an ad. I'm guessing the last two letters of the base64 string tripped it.

I sent an bug report to them, but in the meantime, this might be worth mentioning in the docs to save other users some time when debugging site layouts.

**Comment by nkbt** [comment]
ouch! Thanks for info!

**Comment by geelen** [comment]
Yeah wow, ok. Should probably make emoji classname generation the default then ;)
